config/emendatusenigmatica/material/rftools/dimensional.json

```diff
@@ -0,0 +1,24 @@
+{
+    "id": "dimensional",
+    "source": "modded",
+    "localizedName": "Dimensional Shard",
+    "processedTypes": [
+        "ore"
+    ],
+    "properties": {
+        "materialType": "gem",
```

Comment:
Does this mean we'd be generating a gem with this? Just reminded that these machines are hard coded to using RFTools Dimensional Shards

Reply:
It's just material type for EE - The processed type is just ore, which is the only thing that is getting generated by EE 😄
The drop from the ore is RFTools Dimensional Shards as mentioned below in the file.